lib/features/family/features/reflect/bloc/grateful_cubit.dart

```diff
@@ -142,9 +149,22 @@ class GratefulCubit extends CommonCubit<GratefulUIModel, GratefulCustom> {
     }
   }
 
-  void onRecommendationChosen(int index) {
-    final organisation = _currentRecommendations[index];
-    if (isCurrentProfileChild()) {
+  void onSelectionChanged(int index) {
+    showActsOfService = index == 0;
+    _emitData();
+  }
+
+  void onRecommendationChosen(int index, bool isActOfService) {
```

Comment:
You don't need this isActOfService. You already know via showActsOfservice==true